scripts/error-codes/codes.json

```diff
@@ -102,26 +102,26 @@
-  "105": "%s(...): A valid React element (or null) must be returned. You may have returned undefined, an array or some other invalid object.",
+  "105": "A valid React element (or null) must be returned. You may have returned undefined, an array or some other invalid object.",
   "106": "%s.state: must be set to an object or null",
   "107": "%s.getChildContext(): childContextTypes must be defined in order to use getChildContext().",
   "108": "%s.getChildContext(): key \"%s\" is not defined in childContextTypes.",
   "109": "%s.render(): A valid React element (or null) must be returned. You may have returned undefined, an array or some other invalid object.",
   "110": "Stateless function components cannot have refs.",
   "111": "There is no registered component for the tag %s",
   "112": "getNextDescendantID(%s, %s): Received an invalid React DOM ID.",
-  "113": "getNextDescendantID(...): React has made an invalid assumption about the DOM hierarchy. Expected `%s` to be an ancestor of `%s`.",
+  "113": "React has made an invalid assumption about the DOM hierarchy. Expected `%s` to be an ancestor of `%s`.",
   "114": "getFirstCommonAncestorID(%s, %s): Expected a valid React DOM ID: %s",
-  "115": "traverseParentPath(...): Cannot traverse from and to the same ID, `%s`.",
+  "115": "Cannot traverse from and to the same ID, `%s`.",
   "116": "traverseParentPath(%s, %s, ...): Cannot traverse from two IDs that do not have a parent path.",
   "117": "traverseParentPath(%s, %s, ...): Detected an infinite loop while traversing the React DOM ID tree. This may be due to malformed IDs: %s",
   "118": "updateTextContent called on non-empty component.",
-  "119": "addComponentAsRefTo(...): Only a ReactOwner can have refs. You might be adding a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded (details: https://reactjs.org/link/refs-must-have-owner).",
-  "120": "removeComponentAsRefFrom(...): Only a ReactOwner can have refs. You might be removing a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded (details: https://reactjs.org/link/refs-must-have-owner).",
+  "119": "Only a ReactOwner can have refs. You might be adding a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded (details: https://reactjs.org/link/refs-must-have-owner).",
+  "120": "Only a ReactOwner can have refs. You might be removing a ref to a component that was not created inside a component's `render` method, or you have multiple copies of React loaded (details: https://reactjs.org/link/refs-must-have-owner).",
   "121": "performUpdateIfNecessary: Unexpected batch number (current %s, pending %s)",
-  "122": "%s(...): Expected the last optional `callback` argument to be a function. Instead received: %s.",
```

Comment:
Since this shifts where the second argument is, I'm going to give this a new error code.